### ISSUE TITLE
Loading screen problem

### ISSUE THREAD

**Issue by slowrix** [yuliskov/SmartYouTubeTV#294]
i bought a tv box 3 days ago and installa a smart youtube everything works well since yesterday I have a problem with some videos where I have the loading text all the time anyone know how to fix or wait to update ?

**Comment by yuliskov** [comment]
@slowrix hi, try this update: https://github.com/yuliskov/SmartYouTubeTV/releases/download/6.17.01/SYTV_Orig_v6.17.01_r.apk

**Comment by slowrix** [comment]
i have this update and not work still and some videos not load

**Comment by yuliskov** [comment]
ok, post the log, please (Boot screen > Tweaks > Save log to file)

**Comment by slowrix** [comment]
i cant find it where?

**Comment by yuliskov** [comment]
When you run the app first time you could see the launcher's select screen. Tweaks option should be there.

**Comment by slowrix** [comment]
I marked unplayable video fix and works :)

**Comment by slowrix** [comment]
I noticed that in my subscriptions I have empty tiles in some places
![IMG_20190531_204802](https://user-images.githubusercontent.com/51240938/58727733-6cf11880-83e5-11e9-8818-a9dd6f1348ac.jpg)

**Comment by slowrix** [comment]
Its not work again.. here log
[log.txt](https://github.com/yuliskov/SmartYouTubeTV/files/3242546/log.txt)

**Comment by SocietasEvanescentes** [comment]
I can confirm this issue.

**Comment by yuliskov** [comment]
@slowrix appreciate your support. If you don't mind please send me the log from this release too.
https://github.com/yuliskov/SmartYouTubeTV/releases/download/6.17.02/SYTV_Orig_v6.17.02_r.apk

**Comment by yuliskov** [comment]
Update: https://github.com/yuliskov/SmartYouTubeTV/releases/download/6.17.03/SYTV_Orig_v6.17.03_r.apk

**Comment by slowrix** [comment]
what is it fixing?

**Comment by yuliskov** [comment]
hope to fix Loading screen problem

**Comment by tuespazio** [comment]
I have the same problems with videos, but I notice something strange.
 I have YouTube red ( premium), if I use My another account without YouTube subscription all works well

**Comment by FaserF** [comment]
Same issue for me on my Xiaomi Mi Box S - Config: 4K Main (and also tried 4K alt)
The strange thing is that all videos work except for music videos?

Tried the latest stable and the new test release from today (6.17.04). Issue persists on both. With the official YT App its working.

**Comment by yuliskov** [comment]
Guys, try to use the latest update. If problem still exists attach here the log.
https://github.com/yuliskov/SmartYouTubeTV/releases/download/6.17.04/SYTV_Orig_v6.17.04_r.apk

**Comment by FaserF** [comment]
> Guys, try to use the latest update. If problem still exists attach here the log.
> https://github.com/yuliskov/SmartYouTubeTV/releases/download/6.17.04/SYTV_Orig_v6.17.04_r.apk

Issue still persists. Here is a log.
[log.txt](https://github.com/yuliskov/SmartYouTubeTV/files/3252643/log.txt)


UPDATE: Now its working again with the latest release. Thank you!

**Comment by saraah99** [comment]
What latest release? Which version? 6.17.04_r didn't do it for me.

**Comment by tuespazio** [comment]
I'm the last version (6.17.04) and still have the same problem with the music videos only keeps on loading screen.

**Comment by saraah99** [comment]
> UPDATE: Now its working again with the latest release. Thank you!

Nope, I did some more tests last night, but the loading problem is still very much there. I tried a whole bunch of different versions.
@FaserF Can you shed more light on what's working again for you?

**Comment by yuliskov** [comment]
Another update: https://github.com/yuliskov/SmartYouTubeTV/releases/download/6.17.05/SYTV_Orig_v6.17.05_r.apk

**Comment by saraah99** [comment]
Just tested 6.17.05, but no luck.

**Comment by slowrix** [comment]
On my phone not working as well music videos
![Screenshot_2019-06-05-22-06-42-278_com liskovsoft videomanager](https://user-images.githubusercontent.com/51240938/58986795-72cc6c80-87de-11e9-8ab1-a04623f8a7fa.png)

**Comment by tuespazio** [comment]
Very strange.
If I switch to guest profile I can see any video also music video and all the video load very faster.
But when take back to my account can't see any music video and all the another videos takes a little bit more to start.

**Comment by slowrix** [comment]
Tuespazio mee to the same if im on guest account IT works

**Comment by d3fz** [comment]
Same loading error for me. 

Fixed it by clearing cache and switching to Lite Main (up to 1080p) launcher. 

Works while logged in (user account), by the way.

**Comment by tuespazio** [comment]
[log.txt](https://github.com/yuliskov/SmartYouTubeTV/files/3259082/log.txt)

Here is my log
Thanks for you work

**Comment by slowrix** [comment]
All video's work on my tv but on my TV only 5 subscriptions are loaded, I still have empty seats ...
![IMG_20190606_133909_1](https://user-images.githubusercontent.com/51240938/59030257-c8495d80-8860-11e9-8890-af6d04c29925.jpg)

**Comment by yuliskov** [comment]
@saraah99 thx for the testing. Try this:
https://github.com/yuliskov/SmartYouTubeTV/releases/download/6.17.06/SYTV_Orig_v6.17.06_r.apk

**Comment by saraah99** [comment]
Nope, still no progress after installing 6.17.06. 

However, it is indeed true that the issue goes away when using the lite main launcher or when logging in as a guest with the pro main launcher.

**Comment by tuespazio** [comment]
Yes I can confirm that the problem still continue after the 6.17.06.
Same behavior if you go to guest account all working flawless

**Comment by yuliskov** [comment]
@tuespazio thx for the help. What about this release? All the same?
https://github.com/yuliskov/SmartYouTubeTV/releases/download/6.17.07/SYTV_Orig_v6.17.07_r.apk

**Comment by ljkk-li** [comment]
I am also unable to load the same problem when I choose 4K

**Comment by saraah99** [comment]
> @tuespazio thx for the help. What about this release? All the same?
> https://github.com/yuliskov/SmartYouTubeTV/releases/download/6.17.07/SYTV_Orig_v6.17.07_r.apk

Same ol', same 'ol, even after clearing the cache and/or logging back in.

**Comment by yuliskov** [comment]
upd:
https://github.com/yuliskov/SmartYouTubeTV/releases/download/6.17.08/SYTV_Orig_v6.17.08_r.apk

**Comment by yuliskov** [comment]
upd:
https://github.com/yuliskov/SmartYouTubeTV/releases/download/6.17.09/SYTV_Orig_v6.17.09_r.apk

**Comment by saraah99** [comment]
Unfortunately, the issue is still there when logged in. I cleared both the cache and data this time and then logged back in, but it didn't make a difference.

**Comment by chayank69** [comment]
Already install the suggested update but still not working.
Almost all music video were affected with this issue.
Stuck in the black loading page for ever.
But other video were good, no problem.

**Comment by yuliskov** [comment]
@chayank69 try this release:
https://github.com/yuliskov/SmartYouTubeTV/releases/download/6.17.10/SYTV_Orig_v6.17.10_r.apk

**Comment by tuespazio** [comment]
@yuliskov for me seems fixed only a messages from Play Protect neves seen that message before.
But working flawless

**Comment by yuliskov** [comment]
@tuespazio ok. Seems that the message isn't related to my app. But to be sure I need to see it. Could you post the screenshot?

**Comment by ljkk-li** [comment]
The problem of not loading was fixed in 6.17.10 , thanks to the developers

**Comment by chayank69** [comment]
Got a notification that there's a beta update, klik upgrade and it bring my music back. Thank u..

**Comment by yuliskov** [comment]
@slowrix I think that your problem is related to some unusual symbols inside channel's name. Please add there the log and I'll try to debug your problem.

**Comment by slowrix** [comment]
I think IT works with last update 6.17.10

**Comment by yuliskov** [comment]
@slowrix so bug with subscriptions is fixed? OK then. Let me know if it appears again.

**Comment by tuespazio** [comment]
@yuliskov here a small video all is working fine.

https://drive.google.com/file/d/1--qDUt9Mtfi4cwlNS9mrOHUICFl3OH3B/view?usp=drivesdk

**Comment by d3fz** [comment]
I can confirm that with the lastest build `6.17.10`, ***the loading problem issue is now fixed***. 

Thanks a lot, @yuliskov.

**Comment by chayank69** [comment]
Yes thank u so much, what ur doin is magic.
Now i can play my favorite music video again.

**Comment by yuliskov** [comment]
Guys, thank you all for the help. This bug couldn't be fixed without your support!

**Comment by tuespazio** [comment]
@yuliskov I don't know if is only me but with the last update 6.17.11 can't playback any video all videos stay on Loading screen

**Comment by f-o** [comment]
I hate to be that guy, but for me, the problem stil occurs on .10 update.

The "LOADING..." will not be stuck for up to 2 minutes, like previously, but the video still stops buffering over and over again during playback.

I can attach a log when I get the chance, but I would like to hear if I should enable/disable certain things for the issue to be resolved.

under Tweaks, there's a "Fix unplayable videos" setting. Should this be on/off?
I can disable "Autoframerate" as well. What do you guys have this set to?

**Comment by f-o** [comment]
[log.txt](https://github.com/yuliskov/SmartYouTubeTV/files/3281847/log.txt)
[log (2).txt](https://github.com/yuliskov/SmartYouTubeTV/files/3281854/log.2.txt)

**Comment by RedTestament** [comment]
Same problem here Im using Version 6.17.11 and can confirm that any video is now in Loading and dont play.

**Comment by f-o** [comment]
Unfortunately it's still occurring, even on the latest releases.

Here's a log. 
The first video I tried gave a long initial loading, and stopped just a few seconds into the playback. 
Error occurred 16:26:19
[log (3).txt](https://github.com/yuliskov/SmartYouTubeTV/files/3297445/log.3.txt)

**Comment by yuliskov** [comment]
upd:
https://github.com/yuliskov/SmartYouTubeTV/releases/download/6.17.18/SYTV_Orig_v6.17.18_r.apk

**Comment by f-o** [comment]
I don't know what's going on, or if I'm doing anything wrong, but please let me know things I could try to help the issue from occurring.

.Heres yet another log:
[log (5).txt](https://github.com/yuliskov/SmartYouTubeTV/files/3310925/log.5.txt)

Issue happened on
16:44:46
16:45:12
16:45:24
16:46:12

The video will stop buffering, and the timestamps I took was when the video stopped playing.
